pkg/controller/resourcegraphdefinition/controller_reconcile.go

```diff
@@ -136,6 +144,11 @@ func (r *ResourceGraphDefinitionReconciler) reconcileResourceGraphDefinitionGrap
 		}
 	}
 
+	// make sure the service accounts configured in the resource graph definition have the required permissions.
+	if err := r.reconcileResourceGraphDefinitionPermissions(ctx, processedRGD.Resources, rgd.Spec.DefaultServiceAccounts); err != nil {
+		return nil, nil, err
+	}
```

Comment:
Do we want a special condition for this? i'm thinking that it shouldn't necessarily be Terminal (as in marking the RGD not Ready) - but more of an advisory condition informing the admins that one of their permissions set isn't configured correctly.

Reply:
sure thing I think thats possible